test/sparse/sparsevector.jl

```diff
@@ -184,6 +184,29 @@ let x = sprand(100, 0.5)
     @test Array(r) == Array(x)[I]
 end
 
+# boolean array index
+let x = sprand(10, 10, 0.5)
+    I = rand(1:size(x, 2), 10)
+    bI = falses(size(x, 2))
+    bI[I] = true
+    @which x[1,bI]
```

Comment:
Probably not. I suspect this was someone (me?) debugging a weird coverage issue...